player/lua/stats.lua

```diff
@@ -985,8 +989,8 @@ local function add_video(s)
             append(s, track["decoder"], {prefix="[", nl="", indent=" ", prefix_sep="",
                    no_prefix_markup=true, suffix="]"})
         end
-        append(s, get_property_cached("hwdec-current"), {prefix="HW:", nl="",
-               indent=o.prefix_sep .. o.prefix_sep,
+        append(s, get_property_cached("hwdec-current", {no=true, [""]=true}),
```

Comment:
Could we instead add getter parameter to append_property to avoid duplicating both filters and filtering logic?